taxi/tests/test_forms.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.forms import AuthenticationForm
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormsTests(TestCase):
+    def test_driver_creation_form_is_valid(self):
+        form_data = {
+            "username": "name_user",
+            "password1": "fgfg123123",
+            "password2": "fgfg123123",
+            "first_name": "Pipi",
+            "last_name": "Kell",
+            "license_number": "AAA12312"
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)
```

Comment:
The `assertEqual` statement comparing `form.cleaned_data` with `form_data` might fail because `cleaned_data` can contain additional fields or transformations not present in `form_data`. Consider checking only the relevant fields or using `assertTrue(form.is_valid())` to ensure form validity.